### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by ciremoussadia**

### CODE REVIEW

**Comment by ciremoussadia** [comment]
@blambeau We just need to add env vars and it's good to go.
Envs to add SLACK_WEBHOOK_URL, DOCKER_HUB_USERNAME & DOCKER_HUB_ACCESS_TOKEN

**Comment by blambeau** [comment]
I can't do the review right now, please keep the klaro card open (and
under your responsibility) on your side (you can add a "blocked" status
if needed in Klaro).

B

On October 30, 2021, GitHub ***@***.***> wrote:
> @blambeau <https://github.com/blambeau> We just need to add env vars
> and it's good to go.
>  Envs to add SLACK_WEBHOOK_URL, DOCKER_HUB_USERNAME &
> DOCKER_HUB_ACCESS_TOKEN
>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/chefbe-dot-net/attitude/pull/5#issuecomment-
955205471>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAAN4ZTXETNCXCCHEXWG3CTUJPT3HANCNFSM5HBBZYTQ>.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+on: push
+  # push:
+  #   branches:
+  #     - master
+
+jobs:
+  # CI:
+  #   uses:
+
+  Tests:
```
Comment (llambeau) [inline .github/workflows/release.yml]: Rename this

**Review by blambeau — CHANGES_REQUESTED** [review]
Make sure tests are executed before push.
Can be merged as soon as the build is green.

**Pull request merged**